### PR TITLE
Dates and breadcrumbs

### PR DESCRIPTION
**Pull request by khandelwal**
I added a breadcrumb to the document landing page, that takes you back to the search results you came from. 

I added a date to the search results page.

### CODE REVIEW

**Comment by khandelwal** [comment]
screen shot 2015-02-18 at 6 55 46 pm

**Comment by khandelwal** [comment]
See breadcrumbs. Also the 'more like this' document titles are forced to title case - some of them are all caps which is obnoxious. 

screen shot 2015-02-18 at 6 56 08 pm

**Comment by khandelwal** [comment]
We'll have to spend some time to figure out how exactly the breadcrumbs should work. But at least we have some limited functionality right now.

File: docusearch/templates/docusearch/detail.html [hunk 1]
```diff
@@ -1,14 +1,17 @@
 {% extends "base.html" %}
 
-
 {% block body %}
 
 <div class="container">
+    <section class="breadcrumbs">
+        <a href="{{url('home')}}">Home &gt;</a>
+        <a href="{{url('doc_search')}}?q={{request.GET.q}}">Search &gt;</a>
```
Comment (geramirez) [inline docusearch/templates/docusearch/detail.html:8]: Should we add the query to the link text when possible? Currently, the user might not know when clicking on `search` will return them to their original search or a blank search.

File: docusearch/templates/search/search.html [hunk 2]
```diff
@@ -40,7 +40,7 @@
         {% for result in page.object_list %}
             <li>
                 <div class="agencies--name">
-                    <h3><a href="{{ result.object.get_absolute_url() }}">{{result.object.title}}</a></h3>
+                    <h3><a href="{{ result.object.get_absolute_url()}}/?q={{request.GET.q}}">{{result.object.title}} ({{result.object.date}})</a></h3>
```
Comment (geramirez) [inline docusearch/templates/search/search.html:43]: Occasionally, the date is missing, we could display something a little smother like ({{result.object.date or "Missing Date"}})

**Comment by khandelwal** [comment]
I think this has now been overcome by events.

**Pull request closed without merging**